src/main/java/com/cloudogu/scm/ci/cistatus/service/CIStatusCollection.java

```diff
@@ -58,6 +59,12 @@ public Stream<CIStatus> stream() {
     return ciStatusList.values().stream();
   }
 
+  public static CIStatusCollection toCollection(Collection<CIStatus> statuses) {
```

Comment:
The plural of "status" is still "status" (or isn't it in English?)

Reply:
Right...